vita3k/gui/CMakeLists.txt

```diff
@@ -40,6 +41,6 @@ add_library(
 	src/trophy_unlocked.cpp
 )
 
-target_include_directories(gui PUBLIC include)
-target_link_libraries(gui PUBLIC host imgui glutil)
+target_include_directories(gui PUBLIC include ${CMAKE_SOURCE_DIR}/vita3k)
```

Comment:
?

Reply:
I have found only3this solution for include interface.h file 🤔

Reply:
Move install_vpk function to app module :)

That way you won't need to include interface.h.

Reply:
i don't want broken this function for drop vpk on vita3k install, keep here